Warning: Prerequisite 'Test::RequiresInternet => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'MALLEN/Test-RequiresInternet-0.05.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'Test::Weaken => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'KRYDE/Test-Weaken-3.022000.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'URI => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'OALDERS/URI-1.76.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::DOM => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'TJMATHER/XML-DOM-1.46.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::DOM::XPath => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'MIROD/XML-DOM-XPath-0.14.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::LibXML => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'SHLOMIF/XML-LibXML-2.0201.tar.gz' with 'writemakefile => NO '/usr/bin/perl Makefile.PL INSTALLDIRS=site' returned status 512'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::LibXML::Reader => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'SHLOMIF/XML-LibXML-2.0201.tar.gz' with 'writemakefile => NO '/usr/bin/perl Makefile.PL INSTALLDIRS=site' returned status 512'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::Parser::PerlSAX => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'KMACLEOD/libxml-perl-0.08.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::SAX => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'GRANTM/XML-SAX-1.02.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::SAX::Base => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'GRANTM/XML-SAX-Base-1.09.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::SAX::Writer => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'PERIGRIN/XML-SAX-Writer-0.57.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::Twig => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'MIROD/XML-Twig-3.52.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::Writer => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'JOSEPHW/XML-Writer-0.625.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'YAML => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'TINITA/YAML-1.29.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
  CJFIELDS/BioPerl-1.7.7.tar.gz
  make -- NOT OK
  No such file or directory
Running install for module 'Bio::SeqIO'
  CJFIELDS/BioPerl-1.7.7.tar.gz
  Has already been unwrapped into directory /root/.cpan/build/BioPerl-1.7.7-0
  CJFIELDS/BioPerl-1.7.7.tar.gz
  Has already been prepared
Running make for C/CJ/CJFIELDS/BioPerl-1.7.7.tar.gz
Warning: Prerequisite 'Data::Stag => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'CMUNGALL/Data-Stag-0.14.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'Graph::Directed => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'JHI/Graph-0.9704.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'HTTP::Request::Common => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'OALDERS/HTTP-Message-6.18.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'HTTP::Response => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'OALDERS/HTTP-Message-6.18.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'IO::Scalar => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'CAPOEIRAB/IO-Stringy-2.112.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'IO::String => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'GAAS/IO-String-1.08.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'IPC::Run => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'TODDR/IPC-Run-20180523.0.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'LWP::UserAgent => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'OALDERS/libwww-perl-6.43.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'List::MoreUtils => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'REHSACK/List-MoreUtils-0.428.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'Module::Build => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'LEONT/Module-Build-0.4229.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'Set::Scalar => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'DAVIDO/Set-Scalar-1.29.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'Test::Memory::Cycle => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'PETDANCE/Test-Memory-Cycle-1.06.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'Test::Most => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'OVID/Test-Most-0.35.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'Test::RequiresInternet => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'MALLEN/Test-RequiresInternet-0.05.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'Test::Weaken => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'KRYDE/Test-Weaken-3.022000.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'URI => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'OALDERS/URI-1.76.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::DOM => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'TJMATHER/XML-DOM-1.46.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::DOM::XPath => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'MIROD/XML-DOM-XPath-0.14.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::LibXML => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'SHLOMIF/XML-LibXML-2.0201.tar.gz' with 'writemakefile => NO '/usr/bin/perl Makefile.PL INSTALLDIRS=site' returned status 512'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::LibXML::Reader => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'SHLOMIF/XML-LibXML-2.0201.tar.gz' with 'writemakefile => NO '/usr/bin/perl Makefile.PL INSTALLDIRS=site' returned status 512'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::Parser::PerlSAX => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'KMACLEOD/libxml-perl-0.08.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::SAX => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'GRANTM/XML-SAX-1.02.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::SAX::Base => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'GRANTM/XML-SAX-Base-1.09.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::SAX::Writer => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'PERIGRIN/XML-SAX-Writer-0.57.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::Twig => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'MIROD/XML-Twig-3.52.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'XML::Writer => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'JOSEPHW/XML-Writer-0.625.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
Warning: Prerequisite 'YAML => 0' for 'CJFIELDS/BioPerl-1.7.7.tar.gz' failed when processing 'TINITA/YAML-1.29.tar.gz' with 'make => NO'. Continuing, but chances to succeed are limited.
  CJFIELDS/BioPerl-1.7.7.tar.gz
  make -- NOT OK
  No such file or directory

    this seems to cause the setup to be stuck for a long time... ##++ FIXME



Running install for module 'Bio::SeqIO::entrezgene'
Fetching with HTTP::Tiny:
http://www.cpan.org/authors/id/C/CJ/CJFIELDS/Bio-ASN1-EntrezGene-1.73.tar.gz
Checksum for /root/.cpan/sources/authors/id/C/CJ/CJFIELDS/Bio-ASN1-EntrezGene-1.73.tar.gz ok
Configuring C/CJ/CJFIELDS/Bio-ASN1-EntrezGene-1.73.tar.gz with Makefile.PL
Warning: prerequisite Bio::Annotation::Comment 0 not found.
Warning: prerequisite Bio::Annotation::DBLink 0 not found.
Warning: prerequisite Bio::Annotation::OntologyTerm 0 not found.
Warning: prerequisite Bio::Annotation::Reference 0 not found.
Warning: prerequisite Bio::Annotation::SimpleValue 0 not found.
Warning: prerequisite Bio::Cluster::SequenceFamily 0 not found.
Warning: prerequisite Bio::Index::AbstractSeq 0 not found.
Warning: prerequisite Bio::Ontology::Term 0 not found.
Warning: prerequisite Bio::Root::Test 0 not found.
Warning: prerequisite Bio::Seq 0 not found.
Warning: prerequisite Bio::SeqFeature::Gene::Exon 0 not found.
Warning: prerequisite Bio::SeqFeature::Gene::GeneStructure 0 not found.
Warning: prerequisite Bio::SeqFeature::Gene::Transcript 0 not found.
Warning: prerequisite Bio::SeqFeature::Generic 0 not found.
Warning: prerequisite Bio::SeqIO 0 not found.
Warning: prerequisite Bio::Species 0 not found.
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Bio::ASN1::EntrezGene
Writing MYMETA.yml and MYMETA.json
  CJFIELDS/Bio-ASN1-EntrezGene-1.73.tar.gz
  /usr/bin/perl Makefile.PL INSTALLDIRS=site -- OK
Running make for C/CJ/CJFIELDS/Bio-ASN1-EntrezGene-1.73.tar.gz
---- Unsatisfied dependencies detected during ----
---- CJFIELDS/Bio-ASN1-EntrezGene-1.73.tar.gz ----
    Bio::Annotation::Comment [requires]
    Bio::Annotation::DBLink [requires]
    Bio::Annotation::OntologyTerm [requires]
    Bio::Annotation::Reference [requires]
    Bio::Annotation::SimpleValue [requires]
    Bio::Cluster::SequenceFamily [requires]
    Bio::Index::AbstractSeq [requires]
    Bio::Ontology::Term [requires]
    Bio::Root::Test [build_requires]
    Bio::Seq [requires]
    Bio::SeqFeature::Gene::Exon [requires]
    Bio::SeqFeature::Gene::GeneStructure [requires]
    Bio::SeqFeature::Gene::Transcript [requires]
    Bio::SeqFeature::Generic [requires]
    Bio::SeqIO [requires]
    Bio::Species [requires]


stuck more than once.  

*^ tin bofh ~ ^**>  sudo strace -p 18660
[sudo] password for tin: 
strace: Process 18660 attached
futex(0x55de4d13a0e8, FUTEX_WAIT, 0, NULL

and just sit there...

**^ tin bofh ~ ^**>  ps -ef | grep  18660
tin      18660 15766  0 07:52 pts/94   00:00:05 docker build -t tin6150/bioperl-ubuntu-1904 -t tin6150/bioperl -f Dockerfile.bioperl-ubuntu-1904 .

-docker build -t tin6150/bioperl-ubuntu-1904 -t tin6150/bioperl -f Dockerfile.bioperl-ubuntu-1904 .
--33*[{docker}]


maybe child process, running test?
but overall takes too long.
and hub.docker never complets, even after 4 hours of build time.
so plan to abandone ubuntu cloud build on docker hub.
maybe just push a local build image to the hub.

**^ tin bofh ~/tin-gh/bioperl ^**>  docker push tin6150/bioperl 
The push refers to repository [docker.io/tin6150/bioperl]
7ef553d6a6e8: Pushed 
5a180ed68994: Pushed 
9fee64cfed1b: Pushed 
d000aa4915bb: Pushed 
41955dd9bcd7: Pushed 
58aeadd04555: Layer already exists 
469beb966f1e: Layer already exists 
8e2b859bc4af: Layer already exists 
3809a3354f7f: Layer already exists 
latest: digest: sha256:2042c6edbb4ba179479aa747cb6ca98ab47ac0c97487f442c74668937e69e0ba size: 2201


